test/test-vapid-helper.js

```diff
@@ -10,7 +10,9 @@ const vapidHelper = require('../src/vapid-helper');
 
 const VALID_AUDIENCE = 'https://example.com';
 const VALID_SUBJECT_MAILTO = 'mailto: example@example.com';
-const VALID_SUBJECT_URL = 'https://exampe.com/contact';
+const VALID_SUBJECT_URL = 'https://example.com/contact';
+const INVALID_SUBJECT_URL_1 = 'http://example.gov';
+const INVALID_SUBJECT_URL_2 = 'ftp://example.net';
```

Comment:
As a test, could you also add localhost mailto and https urls?

Reply:
Sure — but just an FYI, they both should pass, since localhost mailto: URIs are accepted by Apple's push notification server, and localhost https: URIs just generate a console warning but no thrown error, and I don't believe we can test for console output without a bunch of changes to your Mocha test harness.